src/encoded/search_views.py

```diff
@@ -0,0 +1,252 @@
+from snovault.elasticsearch.searches.interfaces import SUMMARY_TITLE
+from snovault.elasticsearch.searches.fields import AuditMatrixWithFacetsResponseField
+from snovault.elasticsearch.searches.fields import AllResponseField
+from snovault.elasticsearch.searches.fields import BasicMatrixWithFacetsResponseField
+from snovault.elasticsearch.searches.fields import BasicSearchResponseField
+from snovault.elasticsearch.searches.fields import BasicSearchWithFacetsResponseField
+from snovault.elasticsearch.searches.fields import BasicReportWithFacetsResponseField
+from snovault.elasticsearch.searches.fields import ClearFiltersResponseField
+from snovault.elasticsearch.searches.fields import ColumnsResponseField
+from snovault.elasticsearch.searches.fields import ContextResponseField
+from snovault.elasticsearch.searches.fields import DebugQueryResponseField
+from snovault.elasticsearch.searches.fields import FiltersResponseField
+from snovault.elasticsearch.searches.fields import IDResponseField
+from snovault.elasticsearch.searches.fields import NotificationResponseField
+from snovault.elasticsearch.searches.fields import NonSortableResponseField
+from snovault.elasticsearch.searches.fields import RawMatrixWithAggsResponseField
+from snovault.elasticsearch.searches.fields import RawSearchWithAggsResponseField
+from snovault.elasticsearch.searches.fields import SearchBaseResponseField
+from snovault.elasticsearch.searches.fields import SortResponseField
+from snovault.elasticsearch.searches.fields import TitleResponseField
+from snovault.elasticsearch.searches.fields import TypeOnlyClearFiltersResponseField
+from snovault.elasticsearch.searches.fields import TypeResponseField
+from snovault.elasticsearch.searches.parsers import ParamsParser
+from snovault.elasticsearch.searches.responses import FieldedResponse
+
+
+def includeme(config):
+    config.add_route('search', '/search{slash:/?}')
+    config.add_route('searchv2_raw', '/searchv2_raw{slash:/?}')
+    config.add_route('searchv2_quick', '/searchv2_quick{slash:/?}')
+    config.add_route('report', '/report{slash:/?}')
+    config.add_route('matrixv2_raw', '/matrixv2_raw{slash:/?}')
+    config.add_route('matrix', '/matrix{slash:/?}')
+    config.add_route('summary', '/summary{slash:/?}')
+    config.add_route('audit', '/audit{slash:/?}')
+    config.scan(__name__)
+
+
+DEFAULT_ITEM_TYPES = [
+    'AntibodyLot',
+    'Award',
+    'Biosample',
+    'BiosampleType',
+    'Dataset',
+    'GeneticModification',
+    'Page',
+    'Pipeline',
+    'Publication',
+    'Software',
+    'Gene',
+    'Target',
+    'File',
+    'Lab'
+]
+
+
+@view_config(route_name='search', request_method='GET', permission='search')
+def search(context, request):
+    # Note the order of rendering matters for some fields, e.g. AllResponseField and
+    # NotificationResponseField depend on results from BasicSearchWithFacetsResponseField.
+    fr = FieldedResponse(
```

Comment:
(Minor)The variable name does not confirm to the snake case naming style and is not meaningful

Reply:
I mean the variable name "fr". It would be hard to know what it means later on in the code. It is not a standard acronym.